init/functions

```diff
@@ -128,6 +128,25 @@ umount_retry() {
     return 0
 }
 
+get_iface_from_mac() {
+    local mac="$1"
+    local iface
+    iface="$(ip -o link | grep -i "$mac" | awk '{print $2}' | cut -d ':' -f1)"
+    echo "$iface"
+}
+
+get_qubes_managed_iface() {
+    local mac
+    local qubes_iface
+    mac="$(qubesdb-read /qubes-mac)"
+    qubes_iface="$(get_iface_from_mac "$mac")"
+    if [ "x$qubes_iface" != "x" ]; then
```

Comment:
Please test if that works. For me it looks like it doesn't: missing `/qubes-mac` results in empty `$mac`, which in turns will result in `grep -i ""` in `get_iface_from_mac`, so it will return all the interfaces...

Reply:
I can add one more check on mac variable but why would this happen?

Reply:
As said above - I think `get_iface_from_mac` called with empty mac will return list of all interfaces, not empty result.

Reply:
Sure, I was just asking if there were a reason/possibility that /qubes-mac path would be empty.